SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -0,0 +1,458 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
This file is almost exclusively moved from SpatialWorkerConnection.cpp except for name changes and where tagged below.